dotcom-rendering/src/components/Discussion/Comments.tsx

```diff
@@ -189,6 +198,20 @@ export const Comments = ({
 		return () => clearTimeout(timer);
 	}, [expanded, comments.length, numberOfCommentsToShow, loadingMore]);
 
+	useEffect(() => {
+		if (expanded && !loadingMore && mobileDiscussionAdsEnabled) {
+			const event = new CustomEvent('comments-loaded');
+			document.dispatchEvent(event);
+		}
+	}, [comments.length, expanded, loadingMore, mobileDiscussionAdsEnabled]);
```

Comment:
Why is `comment.length` in the list of dependencies?

Reply:
Think this was left over from an earlier iteration, I've removed it